imports/plugins/core/system-info/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,21 @@
+extend type Query {
+  systemInformation(shopId: ID!): SystemInformation!
+}
+
+"Represents Reaction Plugin"
+type Plugin {
+  # name of plugin
+  name: String
+  # version of plugin
+  version: String
+}
+
+"Represents Reaction System Infomation"
+type SystemInformation {
+  # core api version
+  apiVersion: String
+  # TODO: data version
+  # TODO: mongo version
+  #plugins installed with name, version information
+  plugins:[Plugin]
+}
```

Comment:
@aldeed thanks for all the attention to detail in here. This part of the code is still mostly unfamiliar to me so my input is pretty cursory. Since this is going into the core API, I think it'd be worth having another set of eyes too. Maybe @mikemurray ?